connection.go

```diff
@@ -654,13 +651,15 @@ func (c *Connection) queueFrame(q *[]frame, f frame) {
 func (c *Connection) queueStreamFrames(pt uint8, protected bool, bareAcks bool) (int, error) {
 	c.log(logTypeConnection, "%v: sendQueuedStreamData pt=%v, protected=%v",
 		c.label(), pt, protected)
-	left := c.mtu
 	frames := make([]frame, 0)
 	sent := int(0)
 	acks := c.recvd.prepareAckRange(protected, false)
 	now := time.Now()
 	txAge := time.Duration(c.retransmitTime) * time.Millisecond
 
+	aeadOverhead :=  c.determineAead(pt).Overhead()
+	left := c.mtu - aeadOverhead - kLongHeaderLength // TODO(ekr@rtfm.com): check header type
```

Comment:
Can you cache this computation so you don't need to repeat it?

Reply:
I'm not entirely sure what you mean. Which repetition of this calculation do you find problematic?

Reply:
You compute left here and then again on line 714